tests/dialects/test_duckdb.py

```diff
@@ -483,6 +483,7 @@ def test_duckdb(self):
             )
 
         self.validate_identity("SELECT ISNAN(x)")
+        self.validate_identity("SELECT ISINF(x)")
```

Comment:
Can you also add a test for transpiling e.g. DuckDB to BigQuery and possibly include other dialects that you know of that support this function?

Reply:
will add a DuckDB<>BigQuery test - I suspect Hive has the same syntax (judging from the `IsNan` special case) but I can't find concrete docs for this online.